include/MixerChannelView.h

```diff
@@ -100,6 +105,8 @@ namespace lmms::gui
 
     private:
         QString elideName(const QString& name);
+        MixerChannel* getMixerChannel() const;
+        bool isMasterChannel() const;
```

Comment:
We generally omit the `get`. Also can probably be inlined.
```suggestion
        auto mixerChannel() const -> MixerChannel* { return Engine::mixer(m_channelIndex); }
        auto isMasterChannel() const -> bool { return m_channelIndex == 0; }
```

Reply:
I have made `isMasterChannel` inline as proposed and have renamed `getMixerChannel` to `mixerChannel`. I did not make the latter inline to keep the dependency to `Engine` hidden. That dependency is no good design anyway and should not be made visible to consumers of the header.